jwql/website/apps/jwql/templates/instrument.html

```diff
@@ -23,48 +31,71 @@ <h1 style="text-align: center">
 
     	<!-- List all possible tools -->
     	<div class="row">
-    		<div class="col-md-6 text-center">
-    			<!-- Images -->
-    			<h2>Images</h2>
-    			<a role="button" class="btn btn-primary my-2 disabled" href={{ url('jwql:unlooked', args=[inst]) }}>Unlooked Images</a>
-				</br>
-    			<a role="button" class="btn btn-primary my-2 disabled" href="#">Image Anomalies</a>
-				</br>
-    			<a role="button" class="btn btn-primary my-2" href={{ url('jwql:archive', args=[inst]) }}>Archived Images</a>
-				</br>
+			<div class="container">
+				<div class="col-md-20 text-center">
+					<h2>Archive</h2>
+					<i class="fas fa-database fa-10x"></i>
+					<hr>
+					<p>View Archived {{ inst }} Images Here</p>
```

Comment:
It looks like this is the only section explanation that has every word capitalized. Maybe the documentation and monitor explanations (lines 59 and 87) could also have this format?

Reply:
If you think so! I can also just write an actual sentence or two. I am trying to think of some clever text that can replace what is currently there :)

Reply:
Yeah, maybe a little more explanation of the options a user would find there would be helpful?

Reply:
@tnking97 Working on this now. I don't want to add too much text to the page because I think the layout is pretty self explanatory. I am thinking of something clever right now 😆